acasclient/lsthing.py

```diff
@@ -19,6 +21,8 @@
 
 
 ROW_NUM_KEY = 'row number'
+ACAS_DDICT = 'ACAS DDICT'
+ACAS_LSTHING = 'ACAS LSTHING'
```

Comment:
I wonder if we should write this as: 🤔 
```python
ACAS_DDICT = ACASDDict.CODE_ORIGIN.upper()
ACAS_LSTHING = ACASLsThingDDict.CODE_ORIGIN.upper()
```

I was thinking on lines of removing duplication in knowledge.